subprojects/gradle/impact-shared/src/main/java/com/avito/impact/configuration/ImplementationConfiguration.kt

```diff
@@ -60,8 +63,7 @@ class ImplementationConfiguration(module: InternalModule) : SimpleConfiguration(
                     .withType(DefaultProjectDependency::class.java)
             }
             .filter {
-                val category = it.attributes.getAttribute(Category.CATEGORY_ATTRIBUTE)?.name
-                category == null || category == Category.LIBRARY
+                it.isUnknown() || it.isLibrary() || it.isPlatform()
```

Comment:
The main change is here - explicit support platform dependency type.

Reply:
I'll try to remove this logic

Reply:
Filtering seems useless.
@dsvoronin , do you remember the problem without this logic? I don't see the reasons.

Reply:
i don't remember, unfortunately